### ISSUE TITLE
Incorrectly Identified System

### ISSUE THREAD

**Issue by bkhall** [tagyoureit/nodejs-poolController-webClient#26]
I have an IntelliTouch i5+3, but this client identifies it as an i5+3S.

My physical setup is a two body system (pool and spa) with shared equipment. The S designation above is for a SINGLE body system, either pool or spa (not both).

So, the webclient only shows me a single body element (for the Pool). It does give me a Spa circuit button and the system does respond to it correctly, but without a proper Spa body element, I cannot operate the Heater properly (from this client).

Running MASTER poolController and MASTER poolController-webClient.
Chrome Browser
Pentair IntelliTouch i5 FW v1.180

![image](https://user-images.githubusercontent.com/1682276/103795949-3ee5a880-500c-11eb-971b-03df58e27124.png)

![PXL_20210106_162453589](https://user-images.githubusercontent.com/1682276/103796152-7bb19f80-500c-11eb-940b-3ed6b0f04a70.jpg)

Page 12 and 13 of the IntelliTouch Owner's Manual show panel image of the the i5 and i5S systems, confirming that my panel is a dual body, shared equipment system.
http://images.inyopools.com/cloud/documents/pentair-controls-intellitouch.pdf

Lastly, ScreenLogic UI shows two bodies:
![image](https://user-images.githubusercontent.com/1682276/103797211-cc75c800-500d-11eb-88e7-041668226766.png)

**Comment by bkhall** [comment]
I realize now that this issue is more likely a bug in the poolController software, rather than the webClient. Let me know if you want me to move it over there.

**Comment by tagyoureit** [comment]
Yes, please do and include a packet capture (see the issue template there) and we can get it addressed.

**Comment by bkhall** [comment]
Well, I don't know why, but I now have two bodies showing properly (pool and spa). In the last two days, I had to reset the breaker to my pool panel, to replace an internal power relay (pump was stuck on).

When I powered it back up and checked the UI, two bodies appeared - awesome. However, it still shows the system as i5+3S. So, I now believe that the only thing wrong in the poolConroller library is a simple labeling issue. I'll go poke around in that code to see what I can find.
